framer/Screen.coffee

```diff
@@ -24,6 +24,15 @@ class ScreenClass extends BaseClass
 	toInspect: ->
 		return "<Screen #{Utils.roundWhole(@width)}x#{Utils.roundWhole(@height)}>"
 
+	# Point Conversion
+
+	convertPointToLayer: (point, layer) =>
+		return Utils.convertPointFromContext(point, layer, false, true)
+
+	convertPointToCanvas: (point) =>
```

Comment:
No need to rebind the function to the current context since you're not using `@` anywhere.